query_math.go

```diff
@@ -153,25 +153,34 @@ func Not(args ...interface{}) Term {
 
 // RandomOpts contains the optional arguments for the Random term.
 type RandomOpts struct {
-	Float interface{} `gorethink:"float,omitempty"`
```

Comment:
Unfortunately this needs to remain as `interface{}` as the following query is technically valid:

```
r.Random(r.RandomOpts{
    Float: r.Expr(true),
})
```